CHANGES.md

```diff
@@ -1,7 +1,11 @@
-# Pending
+## Pending
 
+Testing:
 - [#249] Updated abci_info test to 0.17.0
 
+Serialization:
+- [#248] Refactor serializers library to use modules and give a nicer annotation to structs.
```

Comment:
Add link like in:

https://github.com/informalsystems/tendermint-rs/pull/255/files#diff-8b1c3fd0d4a6765c16dfd18509182f9dR11-R16

Reply:
The link didn't do anything for me, but I updated it based on previous ChangelLog entries. I hope that's what you had in mind.

Reply:
Sorry, my bad. I though github will expand the lines but it doesn't. If you expand manually I see these lines highlighted:
![image](https://user-images.githubusercontent.com/7441766/81422949-01b2d900-9154-11ea-8d96-99d37c70c96a.png)

I'm OK as long as this generates links (and looks ok when rendered). I don't feel strongly about the consistency regarding the style of links.